CHANGELOG.rst

```diff
@@ -5,6 +5,22 @@ Changelog
 Version History
 ===============
 
+.. _v1.4.2:
+
+1.4.2 (2018-08-28)
+------------------
+
+* Fix #222: Enhanced set queries to support filtering by concrete object fields.
+  + A side effect of enablish this, attribute names now have a constraint which prohibit them from
```

Comment:
I think you made a typo. Should 'enablish' be 'enabling'?

Reply:
hahha what is `enablish`?? Yes, thanks for catching that.